### ISSUE TITLE
Request: config options for thirst replenishing items and temperature modifying blocks/armor

### ISSUE THREAD

**Issue by WrongWeekend** [Schoperation/SchopCraft#11]
I believe that this mod would benefit from allowing players to add custom drinks (from harvestcraft and other mods) for thirst quenching as well as blocks or clothing that can raise or lower temperature. Being able to configure the amount of their effects would also be helpful.

**Comment by Schoperation** [comment]
It definitely would benefit. Multiple people have asked about this before. However, I'd have to put this on low priority right now.

**Comment by Mike-U5** [comment]
I agree with this, and also perhaps settings to override the default temperature of certain biomes.
